crowbar_framework/lib/crowbar/backup/export.rb

```diff
@@ -0,0 +1,192 @@
+#
+# Copyright 2011-2013, Dell
+# Copyright 2013-2015, SUSE LINUX GmbH
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+require "chef"
+require "yaml_db"
+
+module Crowbar
+  class Backup
+    class Export < Base
+      attr_accessor :path
+
+      def initialize(path)
+        self.path = path
+      end
+
+      def export
+        clients
+        nodes
+        roles
+        databags
+        db
+        crowbar
+        meta
+      end
+
+      def clients
+        chef(
+          "client",
+          "::Chef::ApiClient"
```

Comment:
That's why I was asking, because knife backup passes the constants directly and doesn't use  ```constantize```